starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -37,13 +37,16 @@ import agasc
 from proseco.core import ACABox
 from proseco.catalog import get_effective_t_ccd
 from proseco.guide import get_imposter_mags
+import proseco.characteristics as char
 
 import mica.stats.acq_stats
 import mica.stats.guide_stats
 
 ACQS = mica.stats.acq_stats.get_stats()
 GUIDES = mica.stats.guide_stats.get_stats()
 
+def _get_aca_limits():
```

Comment:
I've largely been trying to toss things into starcheck.utils but this trivial method seems fine here.

Reply:
Good point, I was drawn in by seeing this stuff here in the same module. Should all of the Python code here just be moved out as well (in a separate PR)?

Reply:
I think I've got a bunch of it in https://github.com/sot/starcheck/pull/327/files so either way.